runtime/cudaq/builder/QuakeValue.h

```diff
@@ -39,6 +40,9 @@ class QuakeValue {
   /// is equal to the number provided as input at runtime.
   bool canValidateVectorNumElements = true;
 
+  std::map<std::size_t, QuakeValue> extractedFromIndex;
+  std::map<void *, QuakeValue> extractedFromValue;
```

Comment:
Do these need to be sorted by key? (`unordered_map`?)

Reply:
Nope, will make the update to unordered_map

Reply:
Looks like I can't since QuakeValue is an incomplete type at this point. map works, but unordered_map does not.

Reply:
Interesting.